src/synopsys-action/synopsys-bridge.ts

```diff
@@ -91,6 +93,23 @@ export class SynopsysBridge {
         cwd: workingDirectory
       }
       try {
+        if (inputs.ENABLE_NETWORK_AIR_GAP) {
+          info('bridge default path :'.concat(this.getBridgeDefaultPath()))
+          info('synopsys_bridge_path:'.concat(inputs.SYNOPSYS_BRIDGE_PATH))
+          //const bridgepath = inputs.SYNOPSYS_BRIDGE_PATH.length == 0 ? this.getBridgeDefaultPath() : inputs.SYNOPSYS_BRIDGE_PATH
+          info('inputs.SYNOPSYS_BRIDGE_PATH.length::'.concat(inputs.SYNOPSYS_BRIDGE_PATH))
+          if (inputs.SYNOPSYS_BRIDGE_PATH.length !== 0) {
+            this.bridgeExecutablePath = await this.setBridgeExecutablePath(osName, inputs.SYNOPSYS_BRIDGE_PATH)
+            if (!checkIfPathExists(this.bridgeExecutablePath)) {
```

Comment:
where are we validating the user provided SYNOPSYS_BRIDGE_PATH, seems like you're checking the executable file path directly